src/effects/effects_apply.c

```diff
@@ -86,9 +86,7 @@ static void ApplyCreateNode
 	//--------------------------------------------------------------------------
 
 	LabelID labels[lbl_count];
-	for(ushort i = 0; i < lbl_count; i++) {
-		fread_assert(labels + i, sizeof(LabelID), stream);
-	}
+	fread_assert(labels, lbl_count * sizeof(LabelID), stream);
```

Comment:
I think sizeof(labels) will do the multiplication

Reply:
you're probably right.